native_client/java/Makefile

```
@@ -12,8 +14,23 @@ clean: apk-clean
 apk-clean:
 	./gradlew clean
 
-apk: apk-clean bindings
-	LIBDEEPSPEECH_SO=$(LIBDEEPSPEECH_SO) ./gradlew build
+libs-clean:
+	rm -fr libdeepspeech/libs/*/libdeepspeech.so
+
+libdeepspeech/libs/arm64-v8a/libdeepspeech.so:
+	mkdir libdeepspeech/libs/arm64-v8a/ || true
```

Consider using `make`'s built-in feature for ignoring command failures, by starting a command with `-`, for example.
```suggestion
	-mkdir libdeepspeech/libs/arm64-v8a/
```
If this change is accepted, please apply it elsewhere (ie below).

---

Thanks but this PR is still WIP, I have not yet finished working on that, nor asked anyone to have a look. It's unlikely this part of the code will stay, at least in this form, because it's likely not such a good idea.